src/data/forecast.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
As above you should use 2 spaces for indentation throughout all your files to keep things consistent